runtimes/routing.py

```
@@ -161,6 +170,11 @@ async def route_and_execute(self, spec: TaskSpec) -> tuple[TaskResult, RoutingDe
             provider_used=spec.provider_preference,
             reason=f"Best-fit runtime for task_type='{task_type}' (tier={runtime.TIER.value})",
         )
+        # Append audit detail to decision.reason for observability
+        try:
+            decision.reason += f"; candidates={candidates_info}"
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)